tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -68,18 +68,19 @@ NHLT_BIN=nhlt-sof-lnl-nocodec-fpga-4ch.bin,PASSTHROUGH=true,DMIC_IO_CLK=19200000
 
 # CAVS HDA topology with mixer-based efx eq pipelines for HDA and passthrough pipelines for HDMI
 "sof-hda-generic\;sof-hda-efx-generic\;HDA_CONFIG=efx,USE_CHAIN_DMA=true,DEEPBUFFER_FW_DMA_MS=100,\
-EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,EFX_DRC_PARAMS=passthrough"
+EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
+EFX_DRC_PARAMS=passthrough,EFX_MULTIBAND_DRC_PARAMS=passthrough"
 
 "sof-hda-generic\;sof-hda-efx-generic-2ch\;\
 HDA_CONFIG=efx,NUM_DMICS=2,PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-fir-generic-2ch.bin,\
 USE_CHAIN_DMA=true,DEEPBUFFER_FW_DMA_MS=100,EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
-EFX_DRC_PARAMS=passthrough"
+EFX_DRC_PARAMS=passthrough,EFX_MULTIBAND_DRC_PARAMS=passthrough"
 
 "sof-hda-generic\;sof-hda-efx-generic-4ch\;\
 HDA_CONFIG=efx,NUM_DMICS=4,PDM1_MIC_A_ENABLE=1,PDM1_MIC_B_ENABLE=1,\
 PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-efx-generic-4ch.bin,USE_CHAIN_DMA=true,\
 DEEPBUFFER_FW_DMA_MS=100,EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
-EFX_DRC_PARAMS=passthrough"
+EFX_DRC_PARAMS=passthrough,EFX_MULTIBAND_DRC_PARAMS=passthrough"
```

Comment:
Will try to add some overall EFX configuration to control add of DRC vs. multiband-DRC, both are not needed in same topology for same endpoint. And also simplified versions, maybe with just IIR, just IIR & FIR. Need to find out how to allow the customization without doing a lot of duplication.